edc-tests/e2e-tests/src/test/java/org/eclipse/tractusx/edc/lifecycle/ParticipantRuntime.java

```diff
@@ -62,30 +59,25 @@ public void afterTestExecution(ExtensionContext context) {
     }
 
     @Override
-    public void beforeAll(ExtensionContext context) throws Exception {
+    public void beforeAll(ExtensionContext context) {
         //only run this once
         super.beforeTestExecution(context);
```

Comment:
I think this class should either override `beforeTestExecution` or extend the `RuntimePerClassMethodExtension`